kernel/fs/fs.c

```diff
@@ -745,10 +739,7 @@ static int fs_vfcntl(file_t fd, int cmd, va_list ap) {
     fs_hnd_t *h = fs_map_hnd(fd);
     int rv;
 
-    if(!h) {
-        errno = EBADF;
```

Comment:
In case you didn't see it in Discord:

Why was this removed? Was it not how POSIX handles it? Screwing up the upstream?